src/services/users.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { User } = require('../models/User');
+
+const getAll = () => User.findAll();
+
+const getById = id => User.findByPk(id);
+
+const add = data => User.create(data);
+
+const remove = async(id) => {
+  await User.destroy({
+    where: { id },
+  });
+};
+
+const update = async(id, data) => {
```

Comment:
Please use a consistent parameter naming convention throughout the code. Replace 'id' with 'userId' to better represent the parameter 🧐